### PR TITLE
Add tutorial that describes compression and async loading.

### PR DESCRIPTION
**Pull request by prideout**
This adds the markdown source but does not update the web site, that will be in a subsequent PR.

page1
page2

### CODE REVIEW

File: web/docs/CMakeLists.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ else()
 
     set(CUSTOM_OUTPUTS)
     set(CUSTOM_INPUTS)
-    foreach (NAME triangle redball)
+    foreach (NAME triangle redball suzanne)
```
Comment (bejado) [inline web/docs/CMakeLists.txt]: In the future, maybe we could read the list of tutorial names in from a file.
```
file(STRINGS TUTORIALS tutorials.txt)
```
will read the contents of `tutorials.txt` into the variable `TUTORIALS`. This way, both Python and Cmake can share the list.

File: web/docs/tutorial_suzanne.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+# Create full-size uncompressed Skybox and IBL
+cmgen -x . --format=ktx --size=256 --extract-blur=0.1 syferfontein_18d_clear_2k.hdr
+```
+
+## Define textured material
+
+You might recall the `filamat` file we generated in the previous tutorial for red plastic. For this
+demo, we'll create a material that uses textures for several parameters.
+
+Create the following textz file call it `textured.mat`. Note that our material definition now
```
Comment (bejado) [inline web/docs/tutorial_suzanne.md]:
```suggestion
Create the following text file called `textured.mat`. Note that our material definition now
```

File: web/docs/CMakeLists.txt (hunk 1 above)
Comment (prideout) [reply web/docs/CMakeLists.txt]: Done.

File: web/docs/tutorial_suzanne.md (hunk 2 above)
Comment (prideout) [reply web/docs/tutorial_suzanne.md]: Done.

**Pull request merged**